src/main/java/uk/co/ramp/covid/simulation/covid/Covid.java

```diff
@@ -206,9 +206,9 @@ public double getSymptomDelay() {
     public double getTransAdjustment() {
     	double transAdjustment = 0.0;
     	if(asymptomatic) transAdjustment = this.asymptomaticTransAdjustment;
-    	else if(phase1 && !this.getIsSymptomatic()) transAdjustment = this.asymptomaticTransAdjustment;
-    	else if(phase1 && this.getIsSymptomatic()) transAdjustment = this.symptomaticTransAdjustment;
-    	else if(phase2 && this.getIsSymptomatic()) transAdjustment = this.symptomaticTransAdjustment;
+    	else if(phase1 && !this.isSymptomatic()) transAdjustment = this.asymptomaticTransAdjustment;
+    	else if(phase1 && this.isSymptomatic()) transAdjustment = this.symptomaticTransAdjustment;
+    	else if(phase2 && this.isSymptomatic()) transAdjustment = this.symptomaticTransAdjustment;
```

Comment:
phase2 should always be symptomatic, so we could drop the second part there. Although, I probably wrote that bit of code in the first place!